packages/shared/src/components/comments/PlaceholderComment.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import { ElementPlaceholder } from '../ElementPlaceholder';
+import classed from '../../lib/classed';
+
+export default function PlaceholderComment(): ReactElement {
+  const Text = classed(ElementPlaceholder, 'h-3 rounded-xl');
```

Comment:
I think we can take this out from the component.

Reply:
Well spotted! We actually didn't need this anymore 👀